### ISSUE TITLE
keyboard support in MDM login screen

### ISSUE THREAD

**Issue by geoffm** [linuxmint/Cinnamon#2723]
it seems that the MDM login screen doesn't allow the use of the keyboard. The user is forced to grab his mouse and go click the user name before entering his password. For efficiency and accessibility reasons, it should be possible to accomplish basic tasks, such as logging in, using the keyboard.
